error_reporting/setup.py

```diff
@@ -51,14 +51,14 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.24.1, < 0.25dev',
+    'google-cloud-core >= 0.25.0, < 0.26dev',
     'google-cloud-logging >= 1.0.0, < 2.0dev',
```

Comment:
Looking at the changes, I'm pretty sure we are safe in leaving the dep wide:  `error_reporting` has had no changes which depend on anything newer in `logging`.